core/src/main/scala/pink/cozydev/lucille/QueryPrinter.scala

```diff
@@ -47,6 +47,10 @@ object QueryPrinter {
           sb.append('(')
           printEachNel(q.qs, " ")
           sb.append(s")@${q.num}")
+        case q: Boost =>
+          sb.append('(')
+          printQ(q.q)
+          sb.append(s")^%.${precision}f".format(q.boost))
```

Comment:
This line feels busy. It would be nice to split it up more, but I'm not sure how.